### DISCUSSION TITLE
Allow to update shared recipe

### DISCUSSION THREAD

**Discussion by PATAPOsha** [Genymobile/feedback#79] [Genymotion SaaS] [answered]
Please, add the ability or permission to update shared recipe.
As an Admin, I want to be able to update all recipes that are shared or created among account of my team.
Saving new recipe takes too long, and then you get a mess with multiple copies of a single recipe.
I think it would be logical to implement such permission on group level.

**Comment by AlexandreDorys** [comment, marked as answer]
Hello,

Thanks for reaching out and providing your suggestion. I have added this feature to our product backlog for future development. However, at this time, I am unable to provide a specific ETA regarding its implementation. Our development roadmap is dynamic, and we are continuously evaluating and prioritizing features. I will keep you posted when this feature is released.

Best regards,
Genymotion
